extensions/aris/src/bigDataCluster/controller/types.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Socket } from 'net';
+import { IncomingHttpHeaders } from 'http';
+
+export interface IEndPointsRequest {
```

Comment:
Please move all these to the controller.ts file. Then only export the ones needed to be publicly consumed.

Reply:
fixed.